pkg/extensions/sync/on_demand.go

```diff
@@ -98,7 +98,8 @@ func syncOneImage(imageChannel chan error, cfg Config, storeController storage.S
 
 		credentialsFile, err = getFileCredentials(cfg.CredentialsFile)
 		if err != nil {
-			log.Error().Err(err).Msgf("couldn't get registry credentials from %s", cfg.CredentialsFile)
+			log.Error().Str("errortype", TypeOf(err)).
```

Comment:
Can you show me an example output?

Reply:
This is what it looked like before
```
{"level":"error","error":"open /invalid/path/to/file: no such file or directory","goroutine":68,"caller":"/home/chofnar/go/src/github.com/project-zot/zot/pkg/extensions/sync/sync.go:543","time":"2022-04-15T17:45:33.8741831+03:00","message":"couldn't get registry credentials from /invalid/path/to/file"}
```
And this is what it looks like now
```
{"level":"error","errortype":"*fs.PathError","error":"open /invalid/path/to/file: no such file or directory","goroutine":68,"caller":"/home/chofnar/go/src/github.com/project-zot/zot/pkg/extensions/sync/sync.go:543","time":"2022-04-15T17:45:33.8741831+03:00","message":"couldn't get registry credentials from /invalid/path/to/file"}
```